### PR TITLE
Adds RabbitMQ test image and tag all images with latest

### PR DESCRIPTION
**Pull request by JorTurFer**
As we have discussed on Slack, this PR creates the rabbitmq test image from scratch, instead using a specific sha from `jeffhollan/rabbitmq-client`. The code is exactly the same, but is inside this repo instead of another repo.
Also this PR tag the images with `latest` (and commit sha, both) to automatically use the latest image during e2e test

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [x] A PR is opened to update the documentation on [our docs repo](https://github.com/kedacore/keda-docs)

### CODE REVIEW

File: e2e/images/build.sh [hunk 1]
```diff
@@ -14,16 +14,17 @@ if [ "$1" == "--push" ]
 then
     for IMAGE_NAME in $(find * -name Dockerfile -exec dirname {} \; | tr '/' '-')
     do
-        docker push ghcr.io/kedacore/tests-$IMAGE_NAME:$IMAGE_TAG
-        docker push docker.io/kedacore/tests-$IMAGE_NAME:$IMAGE_TAG
+        docker image push -a ghcr.io/kedacore/tests-$IMAGE_NAME:
```
Comment (zroubalik) [inline e2e/images/build.sh:17]:
```suggestion
        docker image push -a ghcr.io/kedacore/tests-$IMAGE_NAME
```

**Review by zroubalik — COMMENTED** [review]
LGTM, one typo

**Review by zroubalik — APPROVED** [review]
Thanks for this!

File: e2e/images/build.sh (hunk 1 above)
Comment (JorTurFer) [reply e2e/images/build.sh]: Solved, well seen!

**Comment by JorTurFer** [comment]
> Thanks for this!

you're welcome ^^

**Pull request merged**